app/src/main/java/dev/arkbuilders/rate/presentation/quick/QuickScreen.kt

```diff
@@ -157,17 +164,24 @@ fun QuickScreen(
                 state.noInternet -> NoInternetScreen(viewModel::onRefreshClick)
                 state.initialized.not() -> LoadingScreen()
                 isEmpty -> QuickEmpty(navigator = navigator)
-                else -> Content(
-                    state = state,
-                    onFilterChanged = viewModel::onFilterChanged,
-                    onDelete = viewModel::onDelete,
-                    onClick = viewModel::onShowOptions,
-                    onPin = viewModel::onPin,
-                    onUnpin = viewModel::onUnpin,
-                    onNewCode = {
-                        navigator.navigate(AddQuickScreenDestination(newCode = it))
-                    }
-                )
+                else -> {
+                    ctx.sendBroadcast(
+                        Intent(ctx, QuickPairsWidgetReceiver::class.java).apply {
+                            action = QuickPairsWidgetReceiver.PINNED_PAIRS_REFRESH
+                        }
+                    )
```

Comment:
This looks weird, we will send intent on every recomposition. 
Maybe it would be better to move this to `MainActivity.onStop`?
We will update widget when app is minimized. I think this is what we need.

Reply:
Updated. Working fine now